src/main.py

```
@@ -246,31 +288,95 @@ async def respond_with_bot_message(update: Update) -> None:
+                    read_timeout=TELEGRAM_READ_TIMEOUT,
+                )
+        except TimedOut as e:
+            error("Telegram timeout while sending video. %s", e)
+        except (NetworkError, TelegramError) as e:
+            await update.message.reply_text(f"Error sending video: {str(e)}. Please try again later.")
+
+    # Send the group of videos
+    elif isinstance(video, list):
+        media_group = []
+        for video_file in video:
+            media_group.append(InputMediaVideo(open(video_file, 'rb')))
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)